helm/stacks-blockchain/README.md

```diff
@@ -0,0 +1,136 @@
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 2.12+ or Helm 3.0-beta3+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release` and run the node as a **follower**:
+
+```bash
+$ helm repo add blockstack https://charts.blockstack.xyz
+$ helm install my-release blockstack/stacks-blockchain
+```
+
+To install the chart with the release name `my-release` and run the node as a **miner** using your private key [generated from the instructions on this page](https://docs.blockstack.org/mining):
+
+```bash
+$ helm repo add blockstack https://charts.blockstack.xyz
+$ helm install my-release blockstack/stacks-blockchain --set config.node.miner=true --set config.node.seed="REPLACE-WITH-YOUR-PRIVATE-KEY"
+```
+
+These commands deploy the Stacks Blockchain node on a Kubernetes cluster in the default configuration.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
+
+The following tables lists the configurable parameters of the stacks-blockchain chart and their default values.
+
+| Parameter | Description | Default |
+|-|-|-|
+| nameOverride | String to partially override stacks-blockchain.fullname template (will maintain the release name) | nil |
+| fullnameOverride | String to fully override stacks-blockchain.fullname template | nil |
+| node.image.repository | stacks-blockchain container image repository | blockstack/stacks-blockchain |
+| node.image.pullPolicy | stacks-blockchain container image pull policy | IfNotPresent |
+| node.image.tag | stacks-blockchain container image tag | v23.0.0.10-krypton |
+| node.imagePullSecrets | Docker registry secret names as an array | [] |
+| node.replicaCount | Number of stacks-blockchain nodes to run | 1 |
+| node.rpcPort | Port used for RPC connections | 20443 |
+| node.p2pPort | Port used for P2P connections | 20444 |
+| node.labels | Labels to be added to the node Deployment | {} |
+| node.annotations | Annotations to be added to the node Deployment | {} |
+| node.podAnnotations | Annotations to be added to the node pod(s) | {} |
+| node.podAnnotationConfigChecksum | Add Config Checksum to pod Annotations.<br>This will trigger Rolling Updates on configuration changes. | false |
+| node.podSecurityContext | Security Context policies for the node pod(s) | {} |
+| node.securityContext | Security Context policies for the node container | {} |
+| node.debug | Set to true for verbose logging. Set to false for normal verbosity | false |
+| node.jsonLogging | Set to true to enable json logging. Set to false for normal logging | false |
+| node.command | Overriding this is typically not necessary, unless you wish to run the node in a different way | ["/bin/stacks-node"] |
```

Comment:
Is this present in the case of someone using their own image with the chart? Otherwise, i don't see the need to keep this (or at least expose it as an option to override).

Reply:
We've had to change the "command" field on a couple occasions. It's provided as a means for us to support PBC operations but could be useful for others as well.

Tangentially, I'd like the docker image to have an ENTRYPOINT of `/bin/stacks-node` so the k8s deployment wouldn't need to specify a `command` field. Wasn't planning on looping that change in with this PR though.